File: cpp/alphabet/alphabet.cpp
#include <iostream>
#include <cctype>

using namespace std;

int main()
{
    char ch;

    // prompting for input
    cout << "Enter an alphabet: ";
    cin >> ch;

    // converting to uppercase
    ch = toupper(ch);

    // error-check
    if (ch < 'A' || ch > 'Z')
    {
        cout << "Invalid!" << endl;
        return 1;
    }

    // vowels
    char vowels[] = {'A', 'E', 'I', 'O', 'U'};

    // checking for vowels
    for (char c : vowels)
    {
        if (c == ch )
        {
            cout << "It is a vowel." << endl;
            return 0;
        }
    }

    // if it isn't a vowel
    cout << "It is a consonant." << endl;
    return 0;
}

File: python/alphabet.py
import sys

# Prompting for input
char = input("Enter an alphabet: ")

# in case input is empty
if not char:
    print("Empty!")
    sys.exit(1)    

ch = char[0].upper() # extracting the first character from the input, in uppercase

# checking
if ch >= 'A' and ch <= 'Z':
    if ch in ['A', 'E', 'I', 'O', 'U']:
        print("It is a vowel.")
    else:
        print("It is a consonant.")    
else:
    print("Invalid!")  
